CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to 
+making participation in our project and our community a harassment-free experience for everyone, 
+regardless of age, body size, disability, ethnicity, gender identity and expression, technology choices and 
+level of experience, nationality, personal appearance, race, religion (or lack thereof), or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
```

Comment:
"Other conduct" is not wrong, but it's not an "*Example* of unacceptable behavior"

Reply:
maybe just remove "Examples of" from the caption?